packages/ember/src/get-config.ts

```diff
@@ -19,52 +20,58 @@ interface EmberDocfyConfig extends DocfyConfig {
   remarkHbsOptions?: RemarkHbsOptions;
 }
 
-export default function getDocfyConfig(root: string): EmberDocfyConfig {
+export default async function getDocfyConfig(
```

Comment:
Making this `async` breaks [this](https://github.com/josemarluedke/docfy/blob/main/packages/ember/src/index.ts#L152).

Reply:
I was unsure how to test this myself, and this was a theoretical implementation. Definitely open to suggestions to make it work. Just need to be able to use ESM for the config file.